vagrant/provisioning/host_vars/auth

```diff
@@ -0,0 +1,20 @@
+
+packages_to_install: 
```

Comment:
In my never-ending quest to reduce this codebase down to a single line, could the common packages (nmos-common, mdnsbridge, etc) not be factored out into `defaults`, or an `all` group_vars file or something.. and then you should be able to just add the lists:
```
node_packages: "{{ standard_packages_somewhere_else + node_packages_above }}"
```

Reply:
Done